src/plugins/recordTypes/person/fields.js

```diff
@@ -1223,17 +1229,21 @@ export default (configContext) => {
             },
           },
         },
-        birthDateGroupList: {
+        suppliedBirthDateGroupList: {
           [config]: {
             view: {
               type: CompoundInput,
             },
           },
-          birthDateGroup: {
+          suppliedBirthDateGroup: {
             [config]: {
               messages: defineMessages({
+                fullName: {
+                  id: 'field.persons_common.suppliedBirthDateGroup.fullName',
+                  defaultMessage: 'Supplied birth date',
```

Comment:
This seems inconsistent with all the other supplied fields, e.g. "Gender supplied", "Pronoun supplied". Seems like this should be "Birth date supplied", or the others should be "Supplied gender," etc.

Reply:
I had wanted these fields to be easier to find in the search (similar to the above issue, but they were all together under the birth date fields), so I'll try updating the others to be `Supplied ...` and see how that feels in the advanced search.

Reply:
I ended up going with Birth date supplied, and added on `supplied` to other fields to better group them together e.g. `Birth date supplied use restriction`, `Ethnicity supplied use restriction`. Felt like it made them a bit easier to find when looking for them in the search.